doc/lua_api.txt

```diff
@@ -2590,6 +2591,37 @@ An `InvRef` is a reference to an inventory.
 * `get_location()`: returns a location compatible to `minetest.get_inventory(location)`
     * returns `{type="undefined"}` in case location is not known
 
+### `AreaStore`
+A fast access data structure to store areas, and find areas near a given position or area.
+Every area has a `data` string attribute to store additional information.
```

Comment:
I don't like that you have to use a string.  This will add a serialization/deserialization step, reducing performance.  I think it would be better if the mod just stored its data in a table indexed by the area ID.  Running a lookup on this table would be much faster than deserialization.